### PR TITLE
docs: Add EventLogger to docs

### PR DESCRIPTION
**Pull request by marekweb**
Add EventLogger to the docs site. Co-authored with @akalia25 

- Log `ViewStaticPage` on every page view
  - event arguments:
    - `path` 
- Log `DownloadClick` on clicks on download links (which are defined as links with `data-download-name="name"` where the name is included in the event arguments
  - event arguments:
    - `downloadSource` = `docs`
    - `downloadName`: name of the download link, for example `app-download-linux-deb`
    - `downloadLinkUrl` is the clicked link URL

## Test plan

- Run docs site locally with `pnpm run docsite:serve`
- open docs at http://localhost:5080
- Navigate to pages
- Click on download links on `/app`
- Check the events logged (see network requests to debug)

### CODE REVIEW

**Review by akalia25 — APPROVED** [review]
LGTM! 🚀

File: doc/_resources/assets/docsite.js [hunk 1]
```diff
@@ -117,3 +117,41 @@ document.addEventListener('DOMContentLoaded', () => {
...
+
+// Import Sourcegraph's EventLogger
+const importEventLoggerPromise = import('https://cdn.skypack.dev/@sourcegraph/event-logger')
+
+Promise.all([domContentLoadedPromise, importEventLoggerPromise]).then(([_, { EventLogger }]) => {
+  const eventLogger = new EventLogger('https://sourcegraph.com')
+
+  // Log a Docs page view event
+  const eventArguments = { path: window.location.pathname }
+  console.log('ViewStaticPage', eventArguments) // TODO: remove this console log
```
Comment (st0nebraker) [inline doc/_resources/assets/docsite.js:138]: Just checking, when will we want to remove the consoles?

**Review by st0nebraker — APPROVED** [review]
Great, I see the pings going from the network tab! It's correct I shouldn't see any of the actual data being sent through event logger on the api calls, right? I see them in the consoles, but not the network requests. Just making sure

File: doc/_resources/assets/docsite.js (hunk 1 above)
Comment (marekweb) [reply doc/_resources/assets/docsite.js:138]: I'll remove before merging, thanks for checking 👍

**Comment by marekweb** [comment]
> Great, I see the pings going from the network tab! It's correct I shouldn't see any of the actual data being sent through event logger on the api calls, right? I see them in the consoles, but not the network requests. Just making sure

It's supposed to contain arguments in the request payload, the same arguments that are being console logged. It's not in the request on your end?

Screen Shot 2023-03-10 at 5 29 55 PM

**Comment by st0nebraker** [comment]
> It's supposed to contain arguments in the request payload, the same arguments that are being console logged. It's not in the request on your end?

My bad, I see it now. Was looking at the response not payload 🤦🏻 In this case, looks picture perfect! 🖼️

**Review by nathan-downs — APPROVED** [review]
LGTM!

**Pull request merged**